package/Resources/Translations/zh-Hans/Translation.ini

```
@@ -0,0 +1,1704 @@
+Client:DTAConfig:TSFixTitle=新的兼容性修复; New Compatibility Fix
+Client:DTAConfig:TSFixUninstallFailText=卸载 DTA/TI/TS 兼容性修复失败。返回错误：; Uninstalling DTA/TI/TS Compatibility Fix failed. Returned error:
+Client:DTAConfig:TSFixUninstallFailTitle=卸载兼容性修复失败; Uninstalling Compatibility Fix Failed
+Client:DTAConfig:TSFixUninstallText=DTA/TI/TS 兼容性修复已成功卸载。; The DTA/TI/TS Compatibility Fix has been succesfully uninstalled.
+Client:DTAConfig:TSFixUninstallTitle=兼容性修复已卸载; Compatibility Fix Uninstalled
+Client:DTAConfig:TSUndeployAltKey=按住 Alt 键取消部署单位，而非常规移动命令; Undeploy units by holding Alt key instead of a regular move command
+Client:DTAConfig:TSUseBlackBackgroundChat=游戏内聊天消息使用黑色背景; Use black background for in-game chat messages
+Client:DTAConfig:Uninstall=卸载; Uninstall
+Client:DTAConfig:Unpacking=解压中...; Unpacking...
+Client:DTAConfig:Update=更新; Update
+Client:DTAConfig:UpdateConfirmRequiredText=要启用 {0}，客户端需要在您的游戏目录中下载必要的文件。@@这将额外占用 {1} 的磁盘空间，下载可能需要一些时间，具体取决于您的网络连接速度。下载大小为 {2}。@@下载期间您将无法玩游戏。您希望继续吗？; To enable {0} the Client will need to download the necessary files to your game directory.@@This will take an additional {1} of disk space, and the download may take some time@depending on your Internet connection speed. The size of the download is {2}.@@You will not be able to play during the download. Do you wish to continue?
+Client:DTAConfig:UpdateConfirmRequiredTitle=需要确认; Confirmation Required
+Client:DTAConfig:VoiceVolume=语音音量：; Voice Volume:
+Client:DTAConfig:WindowedMode=窗口模式; Windowed Mode
+Client:DTAConfig:WriteGameRegistry=将游戏安装路径写入 Windows@注册表 (使其可以加入 CnCNet 上@其他游戏的房间); Write game installation path to Windows@Registry (makes it possible to join@other games' game rooms on CnCNet)
+Client:DTAConfig:YRShowHidden=显示隐藏物体; Show Hidden Objects
+Client:Main:AddFavorite=添加收藏; Add Favorite
+Client:Main:AddFriend=添加好友; Add Friend
+Client:Main:AdminRequiredExplanation=您似乎正在从一个受写入保护的目录中运行 {0}。@@为了让 {1} 在受写入保护的目录中正常运行，需要管理员权限。@@也请确保您的杀毒软件没有阻止 {1}。; You seem to be running {0} from a write-protected directory.@@For {1} to function properly when run from a write-protected directory, it needs administrative priveleges.@@Please also make sure that your security software isn't blocking {1}.
+Client:Main:AdminRequiredRestartPrompt=您想以管理员权限重启客户端吗？; Would you like to restart the client with administrative rights?
+Client:Main:AdminRequiredTitle=需要管理员权限; Administrative privileges required
+Client:Main:AdminSuffix=(管理员); (Admin)
+Client:Main:AdvancedOptions=高级选项; Advanced Options
+Client:Main:AISpectatorsNotification=AI玩家不喜欢旁观，他们渴望玩真的！; AI players don't enjoy spectating matches. They want some action!
```

It's better to add space character between a Chinese character and an English alphabet

```suggestion
Client:Main:AISpectatorsNotification=AI 玩家不喜欢旁观，他们渴望玩真的！; AI players don't enjoy spectating matches. They want some action!
```

---

### ✅Done : space added
**This change is not controversial, so it is marked as resolved.**